docs/shared/next-plugin.md

```diff
@@ -0,0 +1,73 @@
+# Next.js Plugin
+
+The Nx Plugin for Next.js contains builders, and schematics for managing Next.js applications, and libraries within an Nx workspace. It provides:
+
+- Scaffolding for creating, building, serving, linting, and testing Next.js applications.
+- Integration with building, serving, and exporting a Next.js application.
+- Integration with React libraries within the workspace.
+
+## Installing the Next.js Plugin
+
+Installing the Nest plugin to a workspace can be done with the following:
+
+```shell script
+yarn add -D @nrwl/next
+```
+
+```shell script
+npm install -D @nrwl/next
+```
+
+## Applications
+
+Generating new applications can be done with the following:
+
+```shell script
+nx generate @nrwl/next:application <name>
+```
+
+This creates the following app structure:
+
+```treeview
+myorg/
+├── apps/
+│   ├── myapp/
+│   │   ├── pages/
+│   │   │   ├── index.css
+│   │   │   └── index.tsx
+│   │   ├── jest.conf.js
+│   │   ├── tsconfig.json
+│   │   ├── tsconfig.spec.json
+│   │   └── .eslintrc
+│   └── myapp-e2e/
+│   │   ├── src/
+│   │   │   ├── integrations/
+│   │   │   │   └── app.spec.ts
+│   │   │   ├── fixtures/
+│   │   │   ├── plugins/
+│   │   │   └── support/
+│   │   ├── cypress.json
+│   │   ├── tsconfig.e2e.json
+│   │   └── .eslintrc
+├── libs/
+├── workspace.json
+├── nx.json
+├── package.json
+├── tools/
+├── tsconfig.json
+└── .eslintrc
+```
+
+## See Also
+
+- [Using Next.js](/react/guides/nextjs)
```

Comment:
Would it make sense to just move this guide into the plugin overview page?  I would only keep it separate if it has a lot of content or a different kind of content.